integration-tests/mock-grpc-server/main.go

```diff
@@ -88,6 +89,9 @@ func (s *signalServer) PushNetworkConnectionInfo(stream sensorAPI.NetworkConnect
 			}
 		}
 
+		for _, networkEndpoint := range networkEndpoints {
```

Comment:
Somewhat mirroring my comment on the other collector PR but should we store this information in the database as well as logging it so that the processing in the collector integration tests is more consistent with other tests?

Reply:
Done